compiler/rustc_session/src/config.rs

```diff
@@ -508,6 +508,7 @@ impl FromStr for SplitDwarfKind {
 pub enum OutputType {
     Bitcode,
     ThinLinkBitcode,
+    ThinBitcode,
```

Comment:
Please add doc comments to Bitcode, ThinLinkBitcode and ThinBitcode to explain what the difference between them is...